subprojects/docs/src/main/resources/header.html

```diff
@@ -165,48 +164,99 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
+                <ul id="learning-the-basics-dependency-management">
+                    <li><a href="core_dependency_management.html">What is Dependency Management?</a></li>
+                    <li><a href="declaring_repositories.html">Declaring Repositories</a></li>
+                    <li><a href="declaring_dependencies.html">Declaring Dependencies</a></li>
+                    <li><a href="library_vs_application.html">Understanding Library and Application Differences</a></li>
+                    <li><a href="viewing_debugging_dependencies.html">Viewing and Debugging Dependencies</a></li>
+                    <li><a href="dependency_resolution.html">Understanding Resolution</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#advanced-techniques" aria-expanded="false" aria-controls="advanced-techniques">Advanced Techniques</a>
-                <ul id="advanced-techniques">
-                    <li><a href="https://guides.gradle.org/using-the-worker-api/">Developing Parallel Tasks</a></li>
-                    <li><a href="test_kit.html">Testing Your Build with TestKit</a></li>
-                    <li><a href="ant.html">Using Ant from Gradle</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#declaring-dependency-versions" aria-expanded="false" aria-controls="declaring-dependency-versions">Declaring Versions</a>
+                <ul id="declaring-dependency-versions">
+                    <li><a href="single_versions.html">Declaring Versions and Ranges</a></li>
+                    <li><a href="rich_versions.html">Declaring Rich Versions</a></li>
+                    <li><a href="dynamic_versions.html">Handling Changing Versions</a></li>
+                    <li><a href="dependency_locking.html">Locking Versions</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Example Gradle Projects</a>
-                <ul id="sample-gradle-builds">
-                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL Samples</a></li>
-                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL Samples</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#controlling-transitive-dependencies" aria-expanded="false" aria-controls="controlling-transitive-dependencies">Controlling Transitives</a>
+                <ul id="controlling-transitive-dependencies">
+                    <li><a href="dependency_constraints.html">Upgrading Versions</a></li>
+                    <li><a href="dependency_downgrade_and_exclude.html">Downgrading and Excluding</a></li>
```

Comment:
Looks like adding "Versions" should still fit the layout.
And it would make it clearer IMO.

Reply:
I tried. It needs two lines then.
We can still do it if you feel strongly about it?

Reply:
That's funny, it already needs two lines for me. Web pages remain a total mystery 🤕